controllers/article-controllers.js

```diff
@@ -0,0 +1,14 @@
+const { fetchArticle } = require("../models/article-models");
+
+const getArticle = (request, response, next) => {
+  const { article_id } = request.params;
+  fetchArticle(article_id)
+    .then((article) => {
+      response.status(200).send({ articles: article });
```

Comment:
Small thing, but this is a single article so `article` would make more sense as a key